pkgs/top-level/all-packages.nix

```diff
@@ -38427,7 +38427,10 @@ with pkgs;
 
   r2mod_cli = callPackage ../games/r2mod_cli { };
 
-  r2modman = callPackage ../games/r2modman { };
+  r2modman = callPackage ../games/r2modman {
+    # Electron 25 has regressions making applications unusable.
+    electron = electron_24;
```

Comment:
Are you sure it's the jump from electron 24->25 causing the issue? Based on a similar issue with webcord https://github.com/NixOS/nixpkgs/issues/254539 the jump from 25->26 causing it instead. If you can reproduce the issue consistently, can you try with electron 25 instead?

Reply:
I cannot reproduce the issue, but you're probably right that it was 26, not 25 that causes the issue.  I had 24 in my head because that's what I pinned Heroic to, just to match what upstream is using (as closely as possible).  I will change it to 25.